cost/aws/s3_storage_policy/aws_s3_bucket_policy_check.pt

```diff
@@ -0,0 +1,268 @@
+  iterate $ds_aws_sanitized_buckets
+  request do
+    auth $auth_aws
+    host val(iter_item, "host")
+    path join(["/", val(iter_item, "bucket_name"),"/"])
+    ignore_status [404]
+    query "intelligent-tiering", ""
+  end
+  result do
+    encoding "xml"
+    field "bucket_name", val(iter_item, "bucket_name")
+    field "creation_date", val(iter_item, "creation_date")
+    field "region", val(iter_item, "region")
+    field "tags", val(iter_item, "tags")
+    field "configuration", xpath(response, "//ListIntelligentTieringConfigurationsResult/IntelligentTieringConfiguration")
+  end
+end
+
+datasource "ds_combined_bucket_info" do
+  run_script $js_combined_bucket_info, $ds_aws_bucket_policies, $ds_aws_bucket_tags_list, $ds_get_caller_identity, $param_exclude_tags
+end
+
+script "js_combined_bucket_info", type: "javascript" do
+  parameters "ds_aws_bucket_policies", "ds_aws_bucket_tags_list", "ds_get_caller_identity", "param_exclude_tags"
+  result "results"
+  code <<-EOF
+  var results = []
+  _.each(ds_aws_bucket_policies, function(bucket){
+    var tagged_bucket = _.first(_.filter(ds_aws_bucket_tags_list, function(item) {
+      if (item.bucket_name == bucket.bucket_name){
+        return item
+      }}
+    ))
+    var is_tagged_match = false;
+    if (tagged_bucket !== undefined){
+      tags = tagged_bucket.tags
+      bucket["tags"] = tags
+      var tagKeyValue=""
+
+      _.each(tags, function(tag){
+        if((tag['tagValue']).length > 0){
+          tagKeyValue = tagKeyValue + ', '+ tag['tagKey']+'='+tag['tagValue']
+        }else{
+          tagKeyValue = tagKeyValue + ', '+ tag['tagKey']
+        }
+        key = tag['tagKey']
+        val = tag['tagValue']
+        _.each(param_exclude_tags, function(exclude_tag){
+          exclude_tag_key  = exclude_tag.split('=')[0]
+          exclude_tag_value = exclude_tag.split('=')[1]
+          if (exclude_tag_key == key){
+            if (exclude_tag_value == val){
+              is_tag_matched = true
+            }
+          }
+        })
+      })
+      bucket["tagKeyValue"] = (tagKeyValue.slice(2))
+    } else {
+      bucket["tags"] = ""
+    }
+    bucket["accountId"] = ds_get_caller_identity[0]['account']
+    results.push(bucket)
+  })
+EOF
+end
+###############################################################################
+# Escalations
+###############################################################################
+
+escalation "esc_email" do
+  automatic true
+  label "Send Email"
+  description "Send incident email"
+  email $param_email
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "aws_oversized_buckets" do
+  validate_each $ds_combined_bucket_info do
+    summary_template "{{ len data }} Buckets without intelligent tiering policies found in AWS Account ID: {{with index data 0}}{{ .accountId }}{{end}}"
```

Comment:
We should probably standardize on the syntax/layout of summary templates.
Example: https://github.com/flexera/policy_templates/blob/master/cost/aws/unused_volumes/aws_delete_unused_volumes.pt#L713